README.md

```diff
@@ -58,6 +79,64 @@ CloudFormation:
 
 Then specify this file in the AWS CloudFormation UI during Create Stack.
 
+Run this stack in us-east-1 using 2 t2.micro instances:
+
+[![LaunchStack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=myPushStack&templateURL=https://s3.amazonaws.com/cloudformation-push-setup/push_server.cf)
+
+**Note**: Before using the Launch Stack button here or below, remember that the
+default crypto-key here is only suitable for testing. Real use should have a
+new unique crypto-key, not this default.
```

Comment:
Part of me wonders if that means we shouldn't provide the Launch Stack Button. It's a big, shiny, candy-like button that you should really think about not pushing.

![185](https://cloud.githubusercontent.com/assets/35755/14868887/fad6fe8e-0c84-11e6-948c-b015e1277266.jpg)